<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>confirmpage</class>
 <widget class="QDialog" name="confirmpage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add xxxx</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>590</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>590</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="widthBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>20.000000000000000</double>
   </property>
  </widget>
  <widget class="QComboBox" name="colorBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="colorBrushBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Pen Color</string>
   </property>
  </widget>
  <widget class="QLabel" name="brushLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Brush Color</string>
   </property>
  </widget>
  <widget class="QComboBox" name="styleBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pen Style</string>
   </property>
  </widget>
  <widget class="QComboBox" name="styleBrushBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="styleBrushLabel">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Brush Style</string>
   </property>
  </widget>
  <widget class="QComboBox" name="joinBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pen Join Style</string>
   </property>
  </widget>
  <widget class="QComboBox" name="capBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pen Cap Style</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pen Width</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>310</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>290</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Text: </string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>420</y>
     <width>3</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="capBox_2">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>220</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>200</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Text Alignment</string>
   </property>
  </widget>
  <widget class="QComboBox" name="joinBox_2">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>170</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>150</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Text Color</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>360</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Text Point Size</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="widthBox_2">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>380</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>20.000000000000000</double>
   </property>
  </widget>
  <widget class="QComboBox" name="capBox_3">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>120</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>100</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Text Font Family</string>
   </property>
  </widget>
  <widget class="QComboBox" name="capBox_4">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>70</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>50</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Text Font Style</string>
   </property>
  </widget>
  <widget class="QComboBox" name="capBox_5">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>20</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>0</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Text Alignment</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addPoint">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add point</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="pointBoxX">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>2000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="pointBoxY">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>360</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>2000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="x">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="y">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QLabel" name="warning">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>351</y>
     <width>211</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>330</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Points</string>
   </property>
  </widget>
  <widget class="QLabel" name="centerLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="length">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>470</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Length</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="lBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>490</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>2000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="rectWidth">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="wBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>0</number>
   </property>
   <property name="maximum">
    <double>2000.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>confirmpage</receiver>
   <slot>onConfirmClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>559</y>
    </hint>
    <hint type="destinationlabel">
     <x>258</x>
     <y>566</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>confirmpage</receiver>
   <slot>onCancelClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>66</x>
     <y>557</y>
    </hint>
    <hint type="destinationlabel">
     <x>89</x>
     <y>566</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addPoint</sender>
   <signal>clicked()</signal>
   <receiver>confirmpage</receiver>
   <slot>onAddClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>48</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>667</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onConfirmClick()</slot>
  <slot>onCancelClick()</slot>
  <slot>onAddClick()</slot>
 </slots>
</ui>
